### ISSUE TITLE
Revise 2.4.10 Section Headings text

### ISSUE THREAD

**Issue by MikeElledge** [w3c/wcag#185]
Text reads: Section headings are used to organize the content.

Issue is that this may be misunderstood to mean that HTML5 section element should be used to organize the content, when "section" is being used in its general sense.

Note 1 states: "Heading" is used in its general sense and includes titles and other ways to add a heading to different types of content.

Suggest that Note 1 be revised to include: "Section" and "heading" are used in their general sense..."

**Comment by alastc** [comment]
I assume 2.4.10 was written before there was a section element in HTML5? The full quote from WCAG2 is:

> **2.4.10 Section Headings:** Section headings are used to organize the content. (Level AAA)
Note 1: "Heading" is used in its general sense and includes titles and other ways to add a heading to different types of content.

As the criteria is asking people to add headings to existing sections (in the general or HTML5 sense), I don't feel that it needs to expand the explanatory note.

However, it isn't a big change to have:

> Note 1: "Section" and "Heading" are used in their general sense and this includes titles and other ways to add a heading to different types of content.

**The key question is:** Do people think that people might be confused by section, so they don't add headings if they are not using HTLM5 sections?

I don't think so, but Mike did, so I feel we need some more input, e.g. a survey question.

For a survey question, the options might be:

- It is not necessary to update the 2.4.10 note, there is not sufficient cause for confusion.
- It is necessary to update the 2.4.10 note, and the proposed wording is good.
- It is necessary to update the 2.4.10 note, and I would suggest certain wording (please add in the comment).

**Comment by awkawk** [comment]
If the desire is to have Note 1 changed, then we can't do that except in post-WCAG 2.0 (e.g. 2.1 or later).

There is a definition of Section, does that help?

We can update the understanding document to clarify this if the group feels that it is needed.

**Comment by alastc** [comment]
I think there is a cost / benefit calculation, if it is easy then we could clarify, if it isn't then leave it...

It is the term Section that would be clarified, it is possible to update the note for that?
https://www.w3.org/TR/WCAG20/#sectiondef 

> **section**
> A self-contained portion of written content that deals with one or more related topics or thoughts
> Note: A section may consist of one or more paragraphs and include graphics, tables, lists and sub-sections.

E.g. adding "Note 2: This does not refer to the HTML element `section`."

Another option would be to update the first paragraph of the understanding document:

> The intent of this Success Criterion is to provide headings for sections of a Web page, when the page is organized into sections. **A section in this context is an area of a page, it does not require the usage of the HTML `section` element.**

**Comment by awkawk** [comment]
We can't update the text in WCAG 2.0.  We can shelf this for a future release or add something to Understanding.

If changing the Understanding doc I would keep it generic rather than specifically talking about the HTML section element.

I would prefer to let this one slide entirely however.  With all of the other work that we are lining up, clarifying the understanding document for a AAA success criteria doesn't seem critical.

**Comment by alastc** [comment]
I'm happy to leave it, is it ok to close?

**Comment by awkawk** [comment]
@MikeElledge raised the issue - Mike, what do you think?

**Comment by MikeElledge** [comment]
Up to you. I'm fine with waiting until WCAG 2.x to clarify it and expand 1.3.1 to include HTML5 regions. 

Mike
 

    On Thursday, May 12, 2016 8:04 AM, Andrew Kirkpatrick <notifications@github.com> wrote:
 

 @MikeElledge raised the issue - Mike, what do you think?—

**Comment by awkawk** [comment]
Deferring for WCAG.next
